src/combinator.ts

```diff
@@ -127,62 +125,7 @@ export const orP = <T>(choices: TypeProxy<T>[]): TypeProxy<T> => (value) => {
   return { success: false, error };
 };
 
-export const or2P = <A, B>(left: TypeProxy<A>, right: TypeProxy<B>): TypeProxy<A | B> => (value) => {
```

Comment:
What do you think about keeping these for now, since they offer backwards compatibility, essentially for free?

Reply:
As discussed irl we're going to move forward without these, to prevent people accidentally using them